### PR TITLE
✨(docker-compose) create a docker-compose config for example

### PR DESCRIPTION
**Pull request by lunika**
To ease the use of our images, we have created a docker-compose configuration.

### CODE REVIEW

File: example/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      - REDIS_URL=redis://redis:6379/0
+    depends_on:
+      - mongodb
+      - redis
+      - api
+      - ui
+      - worker
+
+  api:
+    image: fundocker/learninglocker:v2.6.2
```
Comment (jmaupetit) [inline example/docker-compose.yml:33]: Why not using the `local` tag to use local builds?

File: example/env.d/learninglocker [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Misc
```
Comment (jmaupetit) [inline example/env.d/learninglocker:1]: Why about naming this file `env.d/learninglocker.example`? And adding a make rule to enable (copy) it?

File: example/env.d/nginx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+UI_NODE=ui:3000
```
Comment (jmaupetit) [inline example/env.d/nginx:2]: Same remark about the file name here.

File: example/docker-compose.yml (hunk 1 above)
Comment (lunika) [reply example/docker-compose.yml:33]: Because I want to use a published image

File: example/env.d/learninglocker (hunk 2 above)
Comment (lunika) [reply example/env.d/learninglocker:1]: This docker-compose is here as an example, not to be used in this repo. This is why I created a real docker-compose project. If you want to use it you will copy it in your own project

File: example/docker-compose.yml (hunk 1 above)
Comment (jmaupetit) [reply example/docker-compose.yml:33]: IMO the docker-compose configuration from this repository should stand for the project development purpose only. When developing another project that requires learning locker, you may use official/published images in your services, but not here.

File: example/env.d/learninglocker (hunk 2 above)
Comment (jmaupetit) [reply example/env.d/learninglocker:1]: But we need a docker-compose environment in this repository to test our build/modifications, no?

Comment (jmaupetit) [reply example/env.d/learninglocker:1]: I think we need both.

Comment (lunika) [reply example/env.d/learninglocker:1]: No I don't use it for tests, just for having an example how to use the image and how to build a complete learning locker stack.

Comment (jmaupetit) [reply example/env.d/learninglocker:1]: Ok then. So it needs to be clearly stated in the documentation.

Comment (lunika) [reply example/env.d/learninglocker:1]: It is in an `example` directory, I though it should be enough to understand that it is an... example ?

Comment (jmaupetit) [reply example/env.d/learninglocker:1]: :joy: please, document this part. This is not that obvious for everyone that it should be copied in their own project.

Comment (lunika) [reply example/env.d/learninglocker:1]: Done

**Pull request merged**